src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -99,8 +99,8 @@ public sealed class Options : TransformInputBase
             [Argument(ArgumentType.Multiple, HelpText = "Use stop remover or not.", ShortName = "remover", SortOrder = 4)]
             public bool UsePredefinedStopWordRemover = false;
 
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Casing text using the rules of the invariant culture.", ShortName = "case", SortOrder = 5)]
-            public CaseMode TextCase = TextNormalizingEstimator.Defaults.Mode;
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Casing text using the rules of the invariant culture.", Name="TextCase", ShortName = "case", SortOrder = 5)]
+            public CaseMode CaseMode = TextNormalizingEstimator.Defaults.Mode;
```

Comment:
>CaseMode [](start = 28, length = 8)

does it make sense to call it `TextCaseMode` ?

Reply:
This is `CaseMode` in `TextFeaturizingEstimator`, I'd say `Text` will look a bit redundant.

---
In reply to: [265302608](https://github.com/dotnet/machinelearning/pull/2944#discussion_r265302608) [](ancestors = 265302608)